src/components/HistoryTables/MessageHistory/Detail.tsx

```diff
@@ -165,15 +165,15 @@ export default function MessageDetail(props: MessageDetailProps) {
                 </Line>
               )}
               <hr />
-              <SeeMore onClick={() => setSeeMore(!seeMore)}>
+              <p role='button' onClick={() => setSeeMore(!seeMore)}>
                 Click to see {seeMore ? 'less ↑' : 'more ↓'}
-              </SeeMore>
+              </p>
```

Comment:
Same thing here with google translate API - i might be wrong, but better safe than sorry?

Reply:
You were not :) good to know for future implementations!